src/graph/visitor/VidExtractVisitor.cpp

```diff
@@ -165,12 +165,10 @@ void VidExtractVisitor::visit(RelationalExpression *expr) {
       return;
     }
 
-    auto *listExpr = static_cast<ListExpression *>(expr->right());
```

Comment:
```
DCHECK(rightListValue.isList());
```
Some related situations need to be concerned.

Reply:
> ```
> DCHECK(rightListValue.isList());
> ```
> 
> Some related situations need to be concerned.

Thanks, will add this check.